flight/targets/aq32/board-info/board-info.mk

```diff
@@ -1,12 +1,12 @@
 BOARD_TYPE          := 0x94
 BOARD_REVISION      := 0x01
-BOOTLOADER_VERSION  := 0x87
+BOOTLOADER_VERSION  := 0x88
```

Comment:
I believe it was already 0x88 at next (https://github.com/d-ronin/dRonin/pull/1817/files#diff-66c553901e70b9e52436115708752b2d), shouldn't it be 0x89 now?